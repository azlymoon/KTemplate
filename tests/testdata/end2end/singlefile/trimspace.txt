{% let $ten = "10" %}

{% let $blockvar1 %}
  {{ $ten }}
{% end %}
>{{ $blockvar1 }}<

{% let $blockvar2 -%}
  {{ $ten }}
{% end %}
>{{ $blockvar2 }}<

{% let $blockvar3 %}
  {{ $ten }}
{%- end %}
>{{ $blockvar3 }}<

{% let $blockvar4 -%}
  {{ $ten }}
{%- end %}
>{{ $blockvar4 }}<

{% let $blockvar5 %}
  {{- $ten -}}
{% end %}
>{{ $blockvar5 }}<

{% let $blockvar6 -%}
  {{- $ten -}}
{%- end %}
>{{ $blockvar6 }}<

{% set $blockvar1 %}
  {{ $ten }}
  {{ $ten }}
{% end %}
>{{ $blockvar1 }}<

{% set $blockvar1 %}
  {{ $ten -}}
  {{ $ten }}
{% end %}
>{{ $blockvar1 }}<

{% set $blockvar1 %}
  {{ $ten }}
  {{- $ten }}
{% end %}
>{{ $blockvar1 }}<

{% set $blockvar1 %}
  {{ $ten -}}
  {{- $ten }}
{% end %}
>{{ $blockvar1 }}<

{% set $blockvar1 -%}
  {{ $ten -}}
  {{ $ten -}}
{% end %}
>{{- $blockvar1 }}<

{% set $blockvar1 -%}
  {{- $ten -}}
  {{- $ten -}}
{%- end %}
>{{- $blockvar1 }}<
