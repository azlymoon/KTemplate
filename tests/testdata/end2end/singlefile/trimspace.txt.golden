


>
  10
<


>10
<


>
  10<


>10<


>10<


>10<


>
  10
  10
<


>
  1010
<


>
  1010
<


>
  1010
<


>1010<


>1010<
